### DISCUSSION TITLE
Time Series Classification of Intraday Stock for Trend Estimation

### DISCUSSION THREAD

**Discussion by technocrat13** [alan-turing-institute/sktime#2962] [General]
Hello all,

I have a csv that contains univariate minute data for a stock throughout the day (1440 points) for 277 days, my task is to classify this data into various trends. As the data is unclassified it cannot be used for training(?), as a dry run I trained a HIVECOTEV2 on the ACSF1 database as that database is very similar to the classification I want. 

`hc2 = HIVECOTEV2(time_limit_in_minutes=1)`
`hc2.fit(acsf1_X, acsf1_y)`

After which I tried  `y_pred_stocks = hc2.predict(df)` and that errored with `ValueError: ERROR number of attributes in the train does not match that in the test data`

Admittedly this may not be the correct approach. So my questions come to this:

1. If I have raw data without a training set with the appropriate classifiers how do I classify them?
2. How can I scale a training dataset to span enough timepoints to appropriately predict for the given scenario
3. Is this the correct approach for the kind of data that I have? Do there need to be previously defined classes?
4. Tried a run with the arrow_head dataset that is multiindex like my data is, got the same error

**Comment by technocrat13** [comment]
This was a silly question, I have a time series clustering problem, not a time series classification problem

**Reply by fkiraly** [reply]
What is `y`? Are you certain this is classification or clustering, and not forecasting?

**Reply by technocrat13** [reply]
```y``` should be a list with all the deduced classes, and since my data is unlabeled without a training set I went ahead and applied clustering and got the desired results.
